File: core/src/main/java/com/graqr/threshr/model/redsky/product/NutritionFacts.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

import java.util.List;

@Serdeable
public record NutritionFacts(
        String ingredients,
        @JsonProperty("nutrition_label_type_code") String nutritionLabelTypeCode,
        @JsonProperty("value_prepared_list") List<ValuePreparedList> valuePreparedList) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/NutritionFacts.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class NutritionFacts(
    val ingredients: String,
    @param:JsonProperty("nutrition_label_type_code") val nutritionLabelTypeCode: String,
    @param:JsonProperty("value_prepared_list") val valuePreparedList: List<ValuePreparedList>
) {}
